src/StaticValidator.php

```diff
@@ -39,7 +41,7 @@ public static function setPluginManager(ValidatorPluginManager $plugins = null)
     public static function getPluginManager()
     {
         if (null === static::$plugins) {
-            static::setPluginManager(new ValidatorPluginManager());
+            static::setPluginManager(new ValidatorPluginManager(new ServiceManager));
```

Comment:
This is a very common pattern repeated across all SM v3 PRs. May the SM should be injected by default?